src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,79 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
no pls no

Reply:
@danylo-onishchuk It's a part of original task code. But ok, I understand you sorrow)